### PR TITLE
Update endpoint-dlp-getting-started.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to navigation

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/6662

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/endpoint-dlp-getting-started.md [hunk 1]
```diff
@@ -119,15 +119,12 @@ In this deployment scenario, you'll onboard devices that have not been onboarded
 
 1. Open the [Microsoft compliance center](https://compliance.microsoft.com).
 
-2. Open the Compliance Center settings page and choose **Onboard devices**.
-
-   > [!div class="mx-imgBorder"]
-   > ![enable device management.](../media/endpoint-dlp-learn-about-1-enable-device-management.png)
+2. Choose **settings**  and choose **Device Onboarding**.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/endpoint-dlp-getting-started.md]:
```suggestion
2. Choose **Settings** > **Device onboarding**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@chrfox Please merge this PR. Thanks

**Pull request merged**